FreeRTOS/FreeRTOS
# Update coreSNTP demo file to fix warnings

Description
-----------
This PR updates the `SNTPClientTask.c` file to fix compiler warnings , observed in VS2022. It updates the platform set in the project files.

Test Steps
-----------
Compile the coreSNTP demo in VS2022 , without warnings

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
